share/completions/magick.fish

```diff
@@ -0,0 +1,219 @@
+complete -c magick -o scale -d 'Scale the image [geometry]'
+complete -c magick -o segment -d 'Segment an image [values]'
+complete -c magick -o selective-blur -d 'Selectively blur pixels within a contrast threshold [geometry]'
+complete -c magick -o sepia-tone -d 'Simulate a sepia-toned photo [threshold]'
+complete -c magick -o set -d 'Set an image property [property value]'
+complete -c magick -o shade -d 'Shade the image using a distant light source [degrees]'
+complete -c magick -o shadow -d 'Simulate an image shadow [geometry]'
+complete -c magick -o sharpen -d 'Sharpen the image [geometry]'
+complete -c magick -o shave -d 'Shave pixels from the image edges [geometry]'
+complete -c magick -o shear -d 'Slide one edge of the image along the X or Y axis [geometry]'
+complete -c magick -o sigmoidal-contrast -d 'Lightness rescaling using sigmoidal contrast enhancement [geometry]'
+complete -c magick -o sketch -d 'Simulate a pencil sketch [geometry]'
+complete -c magick -o solarize -d 'Negate all pixels above the threshold level [threshold]'
+complete -c magick -o sort-pixels -d 'sort each scanline in ascending order of intensity'
+complete -c magick -o splice -d 'Splice the background color into the image [geometry]'
+complete -c magick -o spread -d 'Displace image pixels by a random amount [radius]'
+complete -c magick -o strip -d 'Strip image of all profiles and comments'
+complete -c magick -o swirl -d 'Swirl image pixels about the center [degrees]'
+complete -c magick -o threshold -d 'Threshold the image [value]'
+complete -c magick -o thumbnail -d 'Create a thumbnail of the image [geometry]'
+complete -c magick -o tile -d 'Tile image when filling a graphic primitive [filename]' -r
+complete -c magick -o tint -d 'Tint the image with the fill color [value]'
+complete -c magick -o transform -d 'Affine transform image'
+complete -c magick -o transparent -d 'Make this color transparent within the image [color]' -xa '(__fish_complete_convert_options color)'
+complete -c magick -o transpose -d 'Flip image vertically and rotate 90 degrees'
+complete -c magick -o transverse -d 'Flop image horizontally and rotate 270 degrees'
+complete -c magick -o trim -d 'Trim image edges'
+complete -c magick -o type -d 'Image type [type]' -xa '(convert -list type)'
+complete -c magick -o unique-colors -d 'Discard all but one of any pixel color'
+complete -c magick -o unsharp -d 'Sharpen the image [geometry]'
+complete -c magick -o vignette -d 'Soften the edges of the image in vignette style [geometry]'
+complete -c magick -o wave -d 'Alter an image along a sine wave [geometry]'
+complete -c magick -o white-threshold -d 'Force all pixels above the threshold into white [value]'
+complete -c magick -o channel-extract -d 'extract the channels in order'
+complete -c magick -o channel-inject -d 'inject the channels in order'
+complete -c magick -o channel-swap -d 'swap the channels'
+complete -c magick -o affinity 'transform image colors to match this set of colors'
```

Comment:
Missing `-d` here

Reply:
Thanks for the missing `-d` catch. Added it and pushed -ff again.

Reply:
Ouch . sorry; my bad. have pushed to the wrong branch on my fork. re-pushed.